README.md

```diff
@@ -3,10 +3,22 @@ CHT Android App
 
 The cht-android application is a thin wrapper to load the [CHT Core Framework](https://github.com/medic/cht-core/) web application in a webview. This allows the application to be hardcoded to a specific CHT deployment and have a partner specific logo and display name. This app also provides some deeper integration with other android apps and native phone functions that are otherwise unavailable to webapps.
 
+# Android App Bundles
+
+The build script produces multiple AABs for publishing to the **Google Play Store**, so the generated `.aab` files need to be uploaded instead of the `.apk` files if the Play Store require so. Old apps published for the first time before Aug 1,  2021 can be updated with the APK format. For each flavor two bundles are generated, one for each rendering engine: _Webview_ and _Xwalk_.
+
+The AABs are named as follows: `cht-android-{version}-{brand}-{rendering-engine}-release.aab`
+
+| Rendering engine | Android version |
+|------------------|-----------------|
+| `webview`        | 10+             |
+| `xwalk`          | 4.4 - 9         |
 
 # APKs
 
-For compatibility with a wide range of devices the build script produces multiple APKs. The two variables are the instruction set used by the device's CPU, and the supported Android version. When publishing to the Google Play Store upload all APKs and it will automatically choose the right one for the target device. However, when sideloading the application it is essential to pick the correct APK or the application may crash.
+For compatibility with a wide range of devices the build script produces multiple APKs. The two variables are the instruction set used by the device's CPU, and the supported Android version. When sideloading the application it is essential to pick the correct APK or the application may crash.
+
+If the Play Store allows to publish new releases of the app in APK format instead of AAB, upload all APKs and it will automatically choose the right one for the target device.
```

Comment:
Slight grammatical improvement:

```suggestion
If distributing APKs via the Play Store, upload all APKs and it will automatically choose the right one for the target device.
```

Also, does the same apply to webview vs webkit AABs? Should we recommend they upload both and Play Store will pick the right one depending on your android version?

Reply:
Yes, I going to copy and paste this in the AAB section: 2 AABs are generated, one by rendering engine but there are no AABs by instruction set.